tiledb/cloud/notebook.py

```diff
@@ -22,6 +22,14 @@
 CHARACTER_ENCODING = "utf-8"
 
 
+class UploadOption(enum.Enum):
+    """Kind of upload option."""
+
+    FAIL = enum.auto()
+    OVERWRITE = enum.auto()
+    AUTO_INCREMENT = enum.auto()
```

Comment:
Since this isn’t implemented yet, I would leave it out of the enum and just make it a TODO to add in the future.